test/ASTMerge/class-template/test.cpp

```diff
@@ -26,5 +26,5 @@ static_assert(sizeof(X0<int>().getValue(1)) == sizeof(int));
 // CHECK: class-template1.cpp:36:7: note: field 'member' has type 'int' here
 // CHECK: class-template2.cpp:36:9: note: field 'member' has type 'float' here
 
-// CHECK: 7 warnings generated.
```

Comment:
What is the warning that disappeared here?

Reply:
It was `warning: missing CXXRecordDecl in imported DeclContext`

Reply:
We addied this new warning in https://github.com/Ericsson/clang/pull/251 and https://github.com/Ericsson/clang/pull/256